ovs_offload/ovs_offload_lnw_v2.py

```diff
@@ -563,7 +561,7 @@ def build_p4rt_config(vf_list=[], acc_pr_list=[],vm_ip_list=[], host_idpf_intf='
 
 def build_args():
     # Create the top-level parser
-    parser = argparse.ArgumentParser(description='Run Linux networking with OVS Offload')
+    parser = argparse.ArgumentParser(description='Run Linux networking V2 with OVS Offload')
```

Comment:
We've dropped support for LNW V2.

Reply:
We have 3 versions of the script to support the OVS offload recipe on all releases from 1.4 to 1.7 as we had customers using the older versions like 1.4 for the POC hence I have retained the 3 versions , but we can remove it in future once customers move to a release 1.7 and above

ovs_offload_lnw.py : (P4:fxp-net_linux-networking.p4, IPU SDK Release >= 1.7.0)
ovs_offload_lnw_v3.py : (P4:fxp-net_linux-networking_v3.p4, IPU SDK Release 1.6.0, 1.6.1)
ovs_offload_lnw_v2.py: (P4:fxp-net_linux-networking_v2.p4, IPU SDK Release 1.4.0)

Reply:
Ah. Got it.